### DISCUSSION TITLE
Changing background color based on languages

### DISCUSSION THREAD

**Discussion by bernabeu-a** [zed-industries/zed#10200] [Feature Requests]
Hi,

In a file that contains multiple languages, is it possible to modify the background color based on the language ?

I would like to achieve something similar to this, with here 2 languages, one with the white background and one with the green one:
<img width="516" alt="exemple" src="https://github.com/zed-industries/zed/assets/99437435/a094b558-c53e-4e17-8880-6f9394f3ca05">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
